### PR TITLE
NC003 Add Electronics smoke test

### PR DESCRIPTION
**Pull request by bhavika48**
Electronics smoke test

### CODE REVIEW

File: src/test/java/com/nopcommerce/demo/cucumber/steps/ElectronicsSteps.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.nopcommerce.demo.cucumber.steps;
+
+import com.nopcommerce.demo.pages.ElectronicsPage;
+import cucumber.api.java.en.Then;
+import cucumber.api.java.en.When;
+
+public class ElectronicsSteps {
+    @When("^I click on electronics link$")
+    public void iClickOnElectronicsLink() {
+        new ElectronicsPage().clickOnElectronicsLink();
```
Comment (HiteshVagQa) [inline src/test/java/com/nopcommerce/demo/cucumber/steps/ElectronicsSteps.java:11]: Can we use homepage ?

File: src/test/java/com/nopcommerce/demo/cucumber/steps/ElectronicsSteps.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    }
+
+    @Then("^I should navigate to electronics page successfully$")
+    public void iShouldNavigateToElectronicsPageSuccessfully() {
+        new ElectronicsPage().getElectronicPageTitle();
```
Comment (ghost) [inline src/test/java/com/nopcommerce/demo/cucumber/steps/ElectronicsSteps.java]: Can we change Home page ?

Comment (HiteshVagQa) [inline src/test/java/com/nopcommerce/demo/cucumber/steps/ElectronicsSteps.java]: can we verify the Electronic page title ?

Comment (ghost) [reply src/test/java/com/nopcommerce/demo/cucumber/steps/ElectronicsSteps.java]: Yes I will

Comment (bhavika48) [reply src/test/java/com/nopcommerce/demo/cucumber/steps/ElectronicsSteps.java]: fixed

**Review by HiteshVagQa — APPROVED** [review]

**Pull request merged**